AudioDeviceExample/ViewController.swift

```diff
@@ -40,23 +44,92 @@ class ViewController: UIViewController {
     let kTextBottomPadding = CGFloat(4)
     let kMaxRemoteVideos = Int(2)
 
+    static let coreAudioDeviceText = "CoreAudio Device"
+    static let engineAudioDeviceText = "AudioEngine Device"
```

Comment:
We should figure out the final names for the devices, perhaps.

`CoreAudioDevice (playback)`
`AVAudioEngineDevice (playback, recording, music)`